### DISCUSSION TITLE
Type map in finetuning

### DISCUSSION THREAD

**Discussion by ZhihaoXu0313** [deepmodeling/deepmd-kit#4051] [Q&A]
Hi,

I have a question about using deepmd/3.0.0b3 for finetuning.

My pretraining model was trained with a full type mapping (the entire periodic table, 118 elements). Should I keep the type_map consistent (full mapping) in the following finetuning step? Or I can re-assign a smaller type_map only for finetune systems?

Hope anyone can share the best practice for this! Thank you!

Best,
ZH

**Comment by njzjz** [comment]
In v3.0.0b3, `type_map` doesn't need to remain the same when performing the fine-tuning step.
